src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -14,12 +14,50 @@ import {
 } from './lib/tutorials_registry_types';
 import { tutorialSchema } from './lib/tutorial_schema';
 import { builtInTutorials } from '../../tutorials/register';
+import { CustomIntegrationsPluginSetup } from '../../../../custom_integrations/server';
+import { Category, CATEGORY_DISPLAY } from '../../../../custom_integrations/common';
+import { HOME_APP_BASE_PATH } from '../../../common/constants';
+
+const emptyContext = {};
+
+function registerTutorialWithCustomIntegrations(
+  customIntegrations: CustomIntegrationsPluginSetup,
+  provider: TutorialProvider
+) {
+  const tutorial = provider(emptyContext);
+  const allowedCategories: Category[] = (tutorial.integrationBrowserCategories
+    ? tutorial.integrationBrowserCategories.filter((category) => {
+        return CATEGORY_DISPLAY.hasOwnProperty(category);
+      })
+    : []) as Category[];
+
+  customIntegrations.registerCustomIntegration({
+    name: tutorial.id,
+    id: tutorial.name,
```

Comment:
I think these are mixed up? Also what is the difference between `name` and `title`?

Reply:
thx, this is wrong. one is human readable. e.g. `id: emsBoundaries` vs `name: EMS Boundaries`